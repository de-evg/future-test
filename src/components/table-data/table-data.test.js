import React from 'react';
import renderer from "react-test-renderer";
import TableData from "./table-data";

describe(`Render TableData correctly`, () => {
  it(`Should Table render correctly with STRING cellData`, () => {
    const cellData = 'some value'
    const tree = renderer
      .create(
        <TableData
          cellData={cellData}        
        />
      )
      .toJSON();
  
      expect(tree).toMatchSnapshot();
  });

  it(`Should Table render correctly with NUMBER cellData`, () => {
    const cellData = 123;
    const tree = renderer
      .create(
        <TableData
          cellData={cellData}        
        />
      )
      .toJSON();
  
      expect(tree).toMatchSnapshot();
  });
});

