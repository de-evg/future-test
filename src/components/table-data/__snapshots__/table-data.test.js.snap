// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render TableData correctly Should Table render correctly with NUMBER cellData 1`] = `
<td
  className="table__data"
>
  123
</td>
`;

exports[`Render TableData correctly Should Table render correctly with STRING cellData 1`] = `
<td
  className="table__data"
>
  some value
</td>
`;
